packages/services/examples/typescript-browser-with-output/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "name": "typescript-browser-with-output",
+  "dependencies": {
+    "@jupyterlab/outputarea": "^0.8.2",
+    "@jupyterlab/rendermime": "^0.8.2",
+    "@jupyterlab/services": "0.46.0",
+    "es6-promise": "^4.1.0"
+  },
+  "scripts": {
+    "build": "tsc && webpack",
+    "clean": "rimraf build"
+  },
+  "devDependencies": {
+    "rimraf": "^2.5.2",
+    "tslint": "~4.5.0",
+    "typescript": "~2.2.0",
```

Comment:
Thanks Ian,

That has successfully changed the error :). That's positive. Upon locking to typescript 2.4.1 I now get the following:

```
$ yarn build
yarn build v0.27.5
$ tsc && webpack
node_modules/@jupyterlab/services/node_modules/@jupyterlab/coreutils/lib/modeldb.d.ts(265,22): error TS2420: Class 'ModelDB' incorrectly implements interface 'IModelDB'.
  Types of property 'createList' are incompatible.
    Type '(path: string) => IObservableUndoableList<JSONValue>' is not assignable to type '<T extends JSONValue>(path: string) => IObservableUndoableList<T>'.
      Type 'IObservableUndoableList<JSONValue>' is not assignable to type 'IObservableUndoableList<T>'.
        Type 'JSONValue' is not assignable to type 'T'.
          Type 'string' is not assignable to type 'T'.

```